### PR TITLE
11.0 mig pos stock picking invoice link

### PR DESCRIPTION
**Pull request by chienandalu**
Links POS generated stock moves and pickings to its corresponding invoice
lines.

cc @Tecnativa

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
Lgtm.
But i dont see the objective of this module. What is the use case ?

**Comment by chienandalu** [comment]
@legalsylvain Is an extension of `stock_picking_invoice_link` and this module allows to have pos invoices linked to their stock moves and pickings as well

File: pos_stock_picking_invoice_link/models/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    pos_order_line_id = fields.Many2one(
+        comodel_name='pos.order.line',
+        string='Related POS Order Line',
+    )
+
+    @api.model
```
Comment (Tardo) [inline pos_stock_picking_invoice_link/models/stock_move.py:14]: Perhaps can use '@api.model_create_multi' here

**Review by Tardo — CHANGES_REQUESTED** [review]
Looks good! Only a suggestion...

**Comment by legalsylvain** [comment]
@chienandalu, any news about @tardo's suggestion ? Regards

**Review by rafaelbn — APPROVED** [review]
@chienandalu please attend @Tardo comments

In my side :+1:

File: pos_stock_picking_invoice_link/models/stock_move.py (hunk 1 above)
Comment (chienandalu) [reply pos_stock_picking_invoice_link/models/stock_move.py:14]: It'd be great, but this is v11 :wink:

**Pull request closed without merging**